operator/cmd/mesh/testdata/manifest-generate/output/ingressgateway_k8s_settings.golden.yaml

```diff
@@ -498,6 +494,10 @@ spec:
     port: 443
     protocol: TCP
     targetPort: 8443
+  - name: status-port
```

Comment:
This is a breaking change, order matters for AWS I think. It will use the first port for health checks

Reply:
Ok. So...Is `status-port` the only special case to keep in mind or are there similar cases to keep in mind while ordering the ports?

Reply:
Updated the PR to handle 15021 and 15020 specially. (ref - https://github.com/istio/istio/issues/12503)